starter-project-web-vue2/store/index.js

```diff
@@ -1,6 +1,7 @@
 import Vuex from 'vuex'
```

Comment:
It is recommended to do it in a way described in the guide. like adding only one store file with your name. and using that only. don't think we need to use the index store and modules here. everyone does their stores